libs/react/color/src/lib/CustomColorModal.tsx

```diff
@@ -43,14 +59,24 @@ export const CustomColorModal: FC<CustomColorModalProps> = ({ onSave, isOpen, on
                     </Center>
                 </ModalHeader>
                 <ModalBody w="full">
-                    <HexColorPicker
-                        style={{ width: '100%' }}
-                        className="responsive"
-                        color={color}
-                        onChange={(newColor) => {
-                            setColor(newColor);
-                        }}
-                    />
+                    <VStack spacing={6}>
+                        <HexColorPicker
+                            style={{ width: '100%' }}
+                            className="responsive"
+                            color={color}
+                            onChange={(newColor) => {
+                                setColor(newColor);
+                            }}
+                        />
+                        <FormControl w='full'>
+                            <Center w='full'>
+                                <HStack>
+                                    <FormLabel htmlFor="email">Hex color</FormLabel>
+                                    <Input w="28" type="text" value={color} onChange={(e) => setColor(e.target.value)} placeholder="#ffffff" />
```

Comment:
We should set the placeholder to #00ffff to match the initial color state

Reply:
Good suggestion 👍